mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -71,6 +71,16 @@ private[python] class PythonMLLibAPI extends Serializable {
       minPartitions: Int): JavaRDD[LabeledPoint] =
     MLUtils.loadLabeledPoints(jsc.sc, path, minPartitions)
 
+  /**
+   * Loads and serializes vectors saved with `RDD#saveAsTextFile`.
+   * @param jsc Java SparkContext
+   * @param path file or directory path in any Hadoop-supported file system URI
+   * @return serialized vectors in a RDD
+   */
+  def loadVectors(jsc: JavaSparkContext,
```

Comment:
scala style: If this method declaration will fit on 1 line, then please do so. (I think it will.)  Otherwise, the parameters should start in the line below the method name and be indented 4 spaces.